control/tests/sysnorm_test.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for sysnorm module.
+
+Created on Mon Jan  8 11:31:46 2024
+Author: Henrik Sandberg
+"""
+
+import control as ct
+import numpy as np
+import pytest
+
+
+def test_norm_1st_order_stable_system():
+    """First-order stable continuous-time system"""
+    s = ct.tf('s')
+    
+    G1 = 1/(s+1)
+    assert np.allclose(ct.norm(G1, p='inf', tol=1e-9 ), 1.0) # Comparison to norm computed in MATLAB
+    assert np.allclose(ct.norm(G1, p=2), 0.707106781186547) # Comparison to norm computed in MATLAB
```

Comment:
See https://github.com/python-control/python-control/pull/960#discussion_r1446462119

Please don't restrict the tolerance to 1e-9. It will create problems like the one solved in #366.

Reply:
That's maybe a bit coarse. I'd say we start with 1e-6 and see if we get failures, e.g. in distribution packages like the openSUSE rpm or conda after the next release.